advanced_source/dynamic_quantization_tutorial.py

```diff
@@ -188,44 +189,44 @@ def tokenize(self, path):
     print(all_output)
 
 ######################################################################
-# It's no GPT-2, but it looks like the model has started to learn the structure of
-# language!
+# GPT-2는 아니지만 모델이 언어 구조를 배우기 시작한 것으로
+# 보여집니다!
 #
-# We're almost ready to demonstrate dynamic quantization. We just need to define a few more
-# helper functions:
+# 동적 양자화를 시연할 준비가 거의 끝났습니다. 몇 가지 helper 함수를 정의하기만
+# 하면 됩니다:
 
 bptt = 25
 criterion = nn.CrossEntropyLoss()
 eval_batch_size = 1
 
-# create test data set
+# 테스트 데이터셋 만들기
 def batchify(data, bsz):
-    # Work out how cleanly we can divide the dataset into bsz parts.
+    # 데이터셋을 bsz 부분으로 얼마나 깔끔하게 나눌 수 있는지 계산해봅니다.
```

Comment:
```suggestion
    # 데이터셋을 bsz 부분으로 얼마나 깔끔하게 나눌 수 있는지 계산합니다.
```
전체적인 어조와 비슷하게 수정했습니다